src/components/views/avatars/BaseAvatar.tsx

```diff
@@ -99,7 +99,7 @@ const BaseAvatar: React.FC<IProps> = (props) => {
     const {
         name,
         idName,
-        title,
+        title = "",
```

Comment:
Why are we setting an empty title on this?

Reply:
To counter https://github.com/vector-im/element-web/issues/26055, but we'll also deal with it on Compound's side.

Reply:
Is there a correct title we could add (e.g. the user's name?)

Reply:
The previous implementation did not have that `title` set actually. So it was actually not required for me to add.
This is a patch till i fix the downstream Compound bug